test/services/licences/fetch-licence-returns.service.test.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const ReturnLogHelper = require('../../support/helpers/return-log.helper.js')
+
+// Thing under test
+const FetchLicenceReturnsService = require('../../../app/services/licences/fetch-licence-returns.service')
+const LicenceHelper = require('../../support/helpers/licence.helper')
+
+describe('Fetch licence returns service', () => {
+  const licenceId = 'fef693fd-eb6f-478d-9f79-ab24749c5dc6'
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+  })
+
+  describe('when there is no optional data in the model', () => {
```

Comment:
Guessing this is a copy & paste description.

```suggestion
  describe('when the licence has return logs', () => {
```